config/forms/2021_2022.json

```diff
@@ -1134,10 +1134,25 @@
               "header": "",
               "description": "",
               "questions": {
+                "age1_known": {
+                  "header": "Do you know the lead tenant’s age?",
+                  "hint_text": "The 'lead' or 'main' tenant is the person in the household who does the most paid work. If several people do the same paid work, the lead tenant is whoever is the oldest.",
+                  "type": "radio",
+                  "answer_options": {
+                    "0": {
+                      "value": "Yes"
+                    },
+                    "1": {
+                      "value": "No"
+                    }
+                  },
+                  "conditional_for": {
+                    "age1": ["Yes"]
+                  }
+                },
                 "age1": {
-                  "check_answer_label": "Tenant’s age",
-                  "header": "What is the tenant’s age?",
-                  "hint_text": "",
+                  "header": "Do you know the lead tenant’s age?",
```

Comment:
Should this be just `Age` or something like that?

Reply:
For the lead tenant the text has changed:

https://miro.com/app/board/o9J_lqoxxRo=/